frontend/src/components/filter/FilterNationalCenterNameSelect.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { getNationalCenters } from '../../fetchers/nationalCenters';
+import FilterSelect from './FilterSelect';
+import { filterSelectProps } from './props';
+
+export default function FilterNationalCenterNameSelect({
+  onApply,
+  inputId,
+  query,
+  title,
+}) {
+  const [nationalCenters, setNationalCenters] = useState([]);
+
+  useEffect(() => {
+    async function fetchNationalCenterNames() {
+      const { centers } = await getNationalCenters(title);
+      setNationalCenters(centers.map((c) => ({ value: c.name, label: c.name })));
+    }
+
+    fetchNationalCenterNames();
+  }, [setNationalCenters, title]);
```

Comment:
this can be empty, setNationalCenters should not need to be in there

Reply:
updated.